File: public/App/Models/Network/Book.swift
//
//  Book.swift
//  app
//
//  Created by sunho on 2019/02/11.
//  Copyright © 2019 sunho. All rights reserved.
//

import Foundation

class Book: Decodable {
    let id: Int
    let name: String
    let img: String
    let epub: String
}

File: backend/apiserver/book/bookparse/book.go
//
// Copyright © 2019 Sunho Kim. All rights reserved.
//

package bookparse

import (
	"gorani/book/sentencer"
	"io"
)

type (
	Book struct {
		Isbn      string
		Name      string
		Author    string
		Cover     Cover
		Genre     []string
		Sentences []sentencer.Sentence
	}

	Cover struct {
		Reader io.Reader
		Ext    string
	}
)
